shell/platform/darwin/macos/framework/Headers/FlutterAppDelegate.h

```diff
@@ -56,13 +56,13 @@ FLUTTER_DARWIN_EXPORT
 /**
  * The application menu in the menu bar.
  */
-@property(weak, nonatomic) IBOutlet NSMenu* applicationMenu;
+@property(weak, nonatomic, nullable) IBOutlet NSMenu* applicationMenu;
```

Comment:
I missed adding these when I added nullability annotations to the split-out delegate above, which causes warnings about not having full nullability annotation.